src/item/Item.js

```diff
@@ -1803,8 +1803,12 @@ new function() { // Injection scope for various item event handlers
      */
     contains: function(/* point */) {
         // See CompoundPath#_contains() for the reason for !!
-        return !!this._contains(
-                this._matrix._inverseTransform(Point.read(arguments)));
+        if (this._matrix.isInvertible()) {
+            return !!this._contains(
+                    this._matrix._inverseTransform(Point.read(arguments)));
+        } else {
+            return false;
+        }
```

Comment:
Could you shorten this to:

```js
var matrix = this._matrix;
return (
	matrix.isInvertible() &&
	!!this._contains(this._matrix._inverseTransform(Point.read(arguments)))
);
```

Reply:
(the first line is so that minification works more efficiently, because local variables can be minified, while properties can't)

Reply:
Made the change. Didn't know that about the minification too. Thanks.

Reply:
Thanks!